cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -255,56 +255,46 @@
-      {% if js_source | length > 0 %}
-        loadScript('{{ static('js/routes/' + request.path.split('/')[1] + '/single.js') }}');
-      {% endif %}
-    }
+    {% macro template_js() %}
+       {% include 'js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
+    {% endmacro %}
+    {% set js_source = template_js() | trim %}
+    {% if js_source | length > 0 %}
+      a.push( '{{ static('js/routes/' + request.path.split('/')[1] + '/single.js') }}' );
+    {% endif %}
+
     {# Check and include component-level JavaScript. #}
-    function loadScriptComponent(){
-      {% if page and page.media %}
-         {% for js in page.media %}
-           loadScript('{{ static('js/routes/on-demand/' + js) }}');
-         {% endfor %}
-      {% endif %}
-    }
+    {% if page and page.media %}
+       {% for js in page.media %}
+         a.push( '{{ static('js/routes/on-demand/' + js) }}' );
+       {% endfor %}
+    {% endif %}
+
     {# TODO: Remove the below block when all pages
              using this template are moved to Wagtail. #}
-    function loadScriptPage(){
-      {# Check and include page-level JavaScript. #}
-      {% macro page_js() %}
-         {% include 'js/routes/' + request.path[1:] + 'index.js' ignore missing %}
-      {% endmacro %}
-      {% set js_source = page_js() | trim %}
-      {% if js_source | length > 0 %}
-        loadScript('{{ static('js/routes/' + request.path[1:] + 'index.js') }}');
-      {% endif %}
-    }
-
-    if (document.body.parentElement.className.indexOf('no-js')===-1){
-      {# Include site-wide JavaScript. #}
-      loadScript('{{ static('js/routes/common.js') }}', function(){
-        loadScriptTemplate();loadScriptComponent();loadScriptPage();
-      });
-    }
-</script>
+    {# Check and include page-level JavaScript. #}
+    {% macro page_js() %}
+       {% include 'js/routes/' + request.path[1:] + 'index.js' ignore missing %}
+    {% endmacro %}
+    {% set js_source = page_js() | trim %}
+    {% if js_source | length > 0 %}
+      a.push( '{{ static('js/routes/' + request.path[1:] + 'index.js') }}' );
+    {% endif %}
 
-<script>
-//<![CDATA[
-    var usasearch_config = { siteHandle: 'cfpb' };
-    loadScript('https://search.usa.gov/javascripts/remote.loader.js');
-//]]>
+    {# Minified dynamic JavaScript loader that injects a script tag in the head
+      of the page. It also provides a callback parameter. #}
+    var b,c,d=[],f=document.scripts[0];function g(){for(var e;d[0]&&'loaded'==d[0].readyState;)e=d.shift(),e.onreadystatechange=null,f.parentNode.insertBefore(e,f)}for(;b=a.shift();)'async'in f?(c=document.createElement('script'),c.async=!1,c.src=b,document.head.appendChild(c)):f.readyState&&(c=document.createElement('script'),d.push(c),c.onreadystatechange=g,c.src=b);
+  }
+  var usasearch_config = { siteHandle: 'cfpb' };
 </script>
 
+<script async="true" src="https://search.usa.gov/javascripts/remote.loader.js"></script>
```

Comment:
Could this be simplified to:

```html
<script async src="https://search.usa.gov/javascripts/remote.loader.js">
```

Reply:
Good catch, it being a boolean it should have the value of `async` or `async="async"`

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3713/commits/38259316db4aed1a8dca4c8c4f94865c74542fd2.